resources.whatwg.org/standard-shared-with-dev.css

```diff
@@ -232,7 +229,12 @@ c-[il] { color: #000000 } /* Literal.Number.Integer.Long */
  h5 + .mdn { margin: -40px 0 0 0; }
  h6 + .mdn { margin: -40px 0 0 0; }
 
-.mdn div { margin: 0; }
+.mdn > div { margin: 0; padding-bottom: 1px; }
+.mdn > div > .less-than-two-engines-flag { color: red; background-color: #eee; font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif; }
+.mdn > div > .all-engines-flag { color: green; background-color: #eee; }
+.mdn > div > details > .less-than-two-engines-text { color: red; font-weight: bold; margin-top: 0px; padding-bottom: 12px; }
+.mdn > div > details > .all-engines-text { color: green; font-weight: bold; margin-top: 0px; padding-bottom: 12px; }
+.mdn > div:last-of-type details > .all-engines-text, .mdn > div:last-of-type > details > .less-than-two-engines-text { padding-bottom: 0px; margin-bottom: 0px; }
```

Comment:
Nit: it looks like the prevailing style is `0`, not `0px`.

Reply:
ah yeah — changed it to drop the `px`